ixnilatisApp/IxnilatisScreens.js

```diff
@@ -22,6 +24,18 @@ export function getIxnilatisScreens(Stack) {
         name="FormGeneralActiveScreen"
         component={FormGeneralActive}
         options={{headerShown:false}}
+      />,
+      <Stack.Screen
+          key="PrivacyScreen"
+          name="PrivacyScreen"
+         component={getPrivacyScreens}
+         options={{ headerShown: false }}
+      />,
+      <Stack.Screen
+          key="AckScreen"
+          name="AckScreen"
+         component={getPrivacyScreens}
```

Comment:
```suggestion
         component={getAcknowledgmentScreens}
```

Reply:
yes! I fixed it offline but did not commit again